cmake/developer_package/compile_flags/sdl.cmake

```diff
@@ -48,7 +48,7 @@ if(CMAKE_COMPILER_IS_GNUCXX OR OV_COMPILER_IS_CLANG OR OV_COMPILER_IS_INTEL_LLVM
     endif()
 elseif(CMAKE_CXX_COMPILER_ID STREQUAL "MSVC")
     set(OV_C_CXX_FLAGS "${OV_C_CXX_FLAGS} /sdl /guard:cf")
-    set(OV_LINKER_FLAGS "${OV_LINKER_FLAGS} /guard:cf")
+    set(OV_LINKER_FLAGS "${OV_LINKER_FLAGS} /guard:cf /DYNAMICBASE")
```

Comment:
it's enabled by default according to https://learn.microsoft.com/en-us/cpp/build/reference/dynamicbase-use-address-space-layout-randomization?view=msvc-170